rust/agama-lib/src/bootloader/store.rs

```diff
@@ -20,30 +20,36 @@
 
 //! Implements the store for the bootloader settings.
 
+use super::{
+    http_client::{BootloaderHTTPClient, BootloaderHTTPClientError},
+    model::BootloaderSettings,
+};
 use crate::base_http_client::BaseHTTPClient;
-use crate::error::ServiceError;
 
-use super::http_client::BootloaderHTTPClient;
-use super::model::BootloaderSettings;
+// FIXME: should we follow this approach more often?
```

Comment:
I have to say I quite like this Result shortcut. I would say it is quite idiomatic in rust ecosystem.